simtools/data_model/format_checkers.py

```diff
@@ -0,0 +1,25 @@
+"""Custom format checkers for jsonschema validation."""
+
+import astropy.units as u
+import jsonschema
+
+from simtools.utils import names
+
+format_checker = jsonschema.FormatChecker()
+
+
+@format_checker.checks("astropy_unit", raises=ValueError)
```

Comment:
Does it actually raise a ValueError if you except it and return a string?

Reply:
Thanks! I've copied this function from existing code in metadata_model without looking at it. Fixed this now.